Lib/opcode.py

```diff
@@ -6,223 +6,25 @@
 
 
 # Note that __all__ is further extended below
-__all__ = ["cmp_op", "opname", "opmap", "stack_effect", "hascompare",
-           "HAVE_ARGUMENT", "EXTENDED_ARG"]
```

Comment:
Are there any potential compatibility issues for users of the opcode module?

Reply:
My intention is to avoid that - I'm adding all the fields back through import. But they are added to ``__all__`` below, because they are only imported in 3.13 (the build imports this file on older python versions, but doesn't use the fields that are only imported in 3.13. Eventually the build won't use this script anymore and then we can make the import unconditional).